code/game/objects/items/weapons/clown_items.dm

```diff
@@ -238,43 +240,80 @@
 
 	if(isitem(target))
 		var/obj/item/target_item = target
+		if(target_item.current_glue_state != GLUE_STATE_NONE) //Check to see if its glued first.
+			to_chat(user,"<span class='warning'>Its already got glue on it!</span>")
```

Comment:
typo

Reply:
Hm? I can't seem to see it.

Reply:
I assume you mean it should be "It already has glue on it.", which is correct.